### DISCUSSION TITLE
Is a new release coming soon?

### DISCUSSION THREAD

**Discussion by utybo** [MrTJP/ProjectRed#1708] [Q&A] [answered]
Hi! The last stable release of ProjectRed dates back to july 2021, and I'd like to use the new features that were released after july soonish (especially the fix for light drops and the computercraft compatibility)

Are there any plans on releasing a new stable version in the near future, or are beta versions stable enough to be used right now on servers?

**Comment by MrTJP** [comment, marked as answer]
Have not had any major bug reports since then, so it is stable enough for release. Will be doing that in the next few days.

**Reply by utybo** [reply]
Alright, thank you!
